tests/atoms/gaia_apps.js

```diff
@@ -296,8 +297,11 @@ var GaiaApps = {
             function() {
               // wait for the displayed app to have expected manifestURL/origin
               let displayedApp = GaiaApps.getDisplayedApp();
+              console.log('*****n****'+displayedApp.entryPoint);
+              console.log('*******'+displayedApp.manifestURL + '::::' + manifestURL);
+              console.log('*******'+displayedApp.entryPoint + '::::' + entryPoint);
               if (displayedApp.manifestURL) {
```

Comment:
I don't think any apps  have entryPoints except contacts and dialer. In all cases displayedApp.entryPoint should be null otherwise, except contacts and dialer. And that would then be equal to entryPoint, which is also most of the times null.